SC Evo Complete/SCEvo_Core.SC2Mod/Base.SC2Data/GameData/EffectData.xml

```diff
@@ -1,14 +1,14 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Catalog>
-    <CEffectDamage default="1" id="ExplosiveBaseDamageSCBW" parent="DU_WEAP">
-        <AttributeFactor index="Light" value="-0.33"/>
-        <AttributeFactor index="Armored" value="0.33"/>
-        <ShieldFactor value="0.3334"/>
+    <CEffectDamage default="1" id="ExplosiveBaseDamageSCBW" parent="DU_WEAP_MISSILE">
+        <AttributeBonus index="Light" value="-0.0625"/>
+        <AttributeBonus index="Armored" value="0.0625"/>
+        <ShieldBonus value="0.0625"/>
     </CEffectDamage>
-    <CEffectDamage default="1" id="ConcussiveBaseDamageSCBW" parent="DU_WEAP">
-        <AttributeFactor index="Light" value="1"/>
-        <AttributeFactor index="Armored" value="-0.5"/>
-        <ShieldFactor value="1"/>
+    <CEffectDamage default="1" id="ConcussiveBaseDamageSCBW" parent="DU_WEAP_MISSILE">
+        <AttributeBonus index="Light" value="0.0625"/>
+        <AttributeBonus index="Armored" value="-0.0625"/>
+        <ShieldBonus value="0.0625"/>
```

Comment:
tiny placeholder values, not sure what to do with these since they must be overwritten by each child anyway, but they could be helpful for color coding in the editor